flink-formats/flink-protobuf/src/main/java/org/apache/flink/formats/protobuf/deserialize/PbCodegenRowDeserializer.java

```diff
@@ -100,6 +102,85 @@ public String codegen(String resultVar, String pbObjectCode, int indent)
         return appender.code();
     }
 
+    @Override
+    public String codegenSplit(
```

Comment:
@ljw-hit Why we still need original codegen methods which are not splittable? Are there any advantages to use original ones? If we only preserve one way to generate code, it will be simpler to maintain.

Reply:
Not all codegen should be split at the beginning of the design. Only when the code exceeds the threshold will it be split, which can reduce the scope of the affected codegen. I think that providing two interfaces can be flexibly selected according to the situation, and sometimes it is not the best after splitting, for example, if there are some internal connection methods.

Reply:
@ljw-hit Could you explain more about internal connection methods? Though it is more flexible to keep two codegen methods but I cannot see much benefit to do that. Too much duplicate codes also increase the maintenance overhead. Will the performance of original codegen method surpass the splitable methods in some cases?

Reply:
@maosuhan I'm ok with both implementations, would you prefer to keep just one interface? if yes i will change it

Reply:
Yes, let's keep one interface.

Reply:
@maosuhan  hi suhan, i have resolved it